iron-dropdown-scroll-manager.html

```diff
@@ -200,23 +200,53 @@
         this._boundScrollHandler = this._boundScrollHandler ||
           this._scrollInteractionHandler.bind(this);
         // Modern `wheel` event for mouse wheel scrolling:
-        document.addEventListener('wheel', this._boundScrollHandler, true);
+        document.addEventListener('wheel', this._boundScrollHandler, {
+          capture: true,
+          passive: false
+        });
         // Older, non-standard `mousewheel` event for some FF:
-        document.addEventListener('mousewheel', this._boundScrollHandler, true);
+        document.addEventListener('mousewheel', this._boundScrollHandler, {
+          capture: true,
+          passive: false
+        });
         // IE:
-        document.addEventListener('DOMMouseScroll', this._boundScrollHandler, true);
+        document.addEventListener('DOMMouseScroll', this._boundScrollHandler, {
+          capture: true,
+          passive: false
+        });
```

Comment:
[MDN](https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/addEventListener#Browser_compatibility) claims IE and Edge don't support objects as the third argument to `addEventListener`; do these work as expected? Maybe they're coerced to booleans?